File: easy/004_multiple_of_numbers/Main.js
const fs = require('fs'),
      readline = require('readline');

(function main(args){
    var filename = args[2];
    
    var reader = readline.createInterface({
        input: fs.createReadStream(filename),
        output: process.stdout,
        terminal: false
    });

    reader.on('line', function(line){
        if(line !== "") {
            var arr = line.split(',');
            console.log(minMultiple(parseInt(arr[0]), parseInt(arr[1])));
        }
    });

})(process.argv);

function minMultiple(x, n) {
    var temp = n;
    var i = 1;
    while(temp < x) {
        temp = n * i;
        i++;
    }

    return temp
}

File: easy/004_multiple_of_numbers/Main.py
import sys

filename = sys.argv[1]

def minMultiple(x, n):
    temp = n
    i = 1
    while temp < x:
        temp = n * i
        i = i + 1

    return temp

with open(filename) as fp:
    line = fp.readline()
    while line:
        if line.strip() != "":
            arr = line.strip().split(",")
            print minMultiple(int(arr[0]), int(arr[1]))

        line = fp.readline()
